supernode/services/artworkregister/status.go

```diff
@@ -10,21 +10,23 @@ const (
 
 	// Process
 	StatusConnected
-	StatusImageUploaded
+	StatusImageProbed
+	StatusImageAndThumbnailUploaded
 
 	// Final
 	StatusTaskCanceled
 	StatusTaskCompleted
 )
 
 var statusNames = map[Status]string{
-	StatusTaskStarted:   "Task started",
-	StatusPrimaryMode:   "Primary Mode",
-	StatusSecondaryMode: "Secondary Mode",
-	StatusConnected:     "Connected",
-	StatusImageUploaded: "Image Uploaded",
-	StatusTaskCanceled:  "Task Canceled",
-	StatusTaskCompleted: "Task Completed",
+	StatusTaskStarted:               "Task started",
+	StatusPrimaryMode:               "Primary Mode",
+	StatusSecondaryMode:             "Secondary Mode",
+	StatusConnected:                 "Connected",
+	StatusImageProbed:               "Image Probed",
+	StatusImageAndThumbnailUploaded: "Imaged And Thumbnailed Uploaded",
```

Comment:
Thumbnailed -> Thumbnail.
but I think should be ThumbnailDetails or ThumbnailParams or ThumbnailCoordinate?

Reply:
@pjenys0 ThumbnailCoordinate makes sense.